php-checks/src/main/java/org/sonar/php/checks/formatting/ControlStructureSpacingCheck.java

```diff
@@ -49,20 +48,20 @@ private void checkForeachStatement(FormattingStandardCheck formattingCheck, AstN
     AstNode asKeyword = foreachExpr.getFirstChild(PHPKeyword.AS);
     AstNode doubleArrow = foreachExpr.getFirstChild(PHPPunctuator.DOUBLEARROW);
 
-    boolean isSpaceCorrectAs = isSpaceAround(asKeyword, 1 /* space before*/, 1/* space after */);
-    boolean isSpaceCorrectDoubleArrow = doubleArrow == null || isSpaceAround(doubleArrow, 1/* space before*/, 1/* space after */);
+//    boolean isSpaceCorrectAs = isSpaceAround(asKeyword, 1 /* space before*/, 1/* space after */);
+//    boolean isSpaceCorrectDoubleArrow = doubleArrow == null || isSpaceAround(doubleArrow, 1/* space before*/, 1/* space after */);
 
     String keyword = null;
-    if (!isSpaceCorrectAs && isSpaceCorrectDoubleArrow) {
-      keyword = "\"" + asKeyword.getTokenOriginalValue() + "\"";
-    } else if (isSpaceCorrectAs && !isSpaceCorrectDoubleArrow) {
-      keyword = "\"" + doubleArrow.getTokenOriginalValue() + "\"";
-    } else if (!isSpaceCorrectAs && !isSpaceCorrectDoubleArrow) {
-      keyword = "\"" + asKeyword.getTokenOriginalValue() + "\" and \"" + doubleArrow.getTokenOriginalValue() + "\"";
-    }
+//    if (!isSpaceCorrectAs && isSpaceCorrectDoubleArrow) {
+//      keyword = "\"" + asKeyword.getTokenOriginalValue() + "\"";
+//    } else if (isSpaceCorrectAs && !isSpaceCorrectDoubleArrow) {
+//      keyword = "\"" + doubleArrow.getTokenOriginalValue() + "\"";
+//    } else if (!isSpaceCorrectAs && !isSpaceCorrectDoubleArrow) {
+//      keyword = "\"" + asKeyword.getTokenOriginalValue() + "\" and \"" + doubleArrow.getTokenOriginalValue() + "\"";
+//    }
 
     if (keyword != null) {
-      formattingCheck.reportIssue("Put exactly one space after and before " + keyword + " in \"foreach\" statement.", node);
+//      formattingCheck.reportIssue("Put exactly one space after and before " + keyword + " in \"foreach\" statement.", node);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)